backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlDatabaseFieldFactory.java

```diff
@@ -46,12 +51,19 @@ public GraphqlDatabaseFieldFactory() {
         .dataFetcher(
             dataFetchingEnvironment -> {
               String name = dataFetchingEnvironment.getArgument("name");
+              validateSchemaName(name);
               String description = dataFetchingEnvironment.getArgument("description");
               database.createSchema(name, description);
               return new GraphqlApiMutationResult(SUCCESS, "Schema %s created", name);
             });
   }
 
+  private void validateSchemaName(String name) {
```

Comment:
if you want to validate schema names that should happen at schema creation.  Now it happens at a specfici usage and that would be too late to fix it. Am I misunderstanding its purpose here?

Currently, there are more reserved schema names such as 'api', 'public_html'. Better centralize those in contects of Schema.java/SchemaMetadata.java